.github/workflows/reviewdog.yml

```diff
@@ -31,4 +31,5 @@ jobs:
       - name: golangci-lint
         uses: ./.github/actions/review-dog
         with:
-          golangci_lint_flags: "--enable-all --timeout=10m --exclude-use-default=false"
+#          golangci_lint_flags: "--enable-all --timeout=10m --exclude-use-default=false"
```

Comment:
Remove commented codes

Reply:
ok, this is a test pr, i want to try how to use the default flags